package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "js-wheatherapp",
+  "version": "1.0.0",
+  "description": "Microverse Fifth JS Project",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
Your project can benefit from having more scripts. It's best practice to add scripts for both `build` and `developer` environments. By doing that you can also inform webpack in which mode should run.
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
  },
```